javascript/ql/test/library-tests/Modules/tests.expected

```diff
@@ -69,19 +69,18 @@ test_Imports
 | tst.html:5:3:5:20 | import f from 'a'; | tst.html:5:17:5:19 | 'a' | 1 |
 | unresolved.js:1:1:1:18 | import f from 'a'; | unresolved.js:1:15:1:17 | 'a' | 1 |
 test_Module_exports
-| a.js:1:1:5:32 | <toplevel> | default | a.js:1:1:3:1 | export  ... n 23;\\n} |
-| a.js:1:1:5:32 | <toplevel> | x | a.js:5:1:5:32 | export  ...  } = o; |
-| a.js:1:1:5:32 | <toplevel> | y | a.js:5:1:5:32 | export  ...  } = o; |
```

Comment:
Looks like we have lost the `y` from this line:
```js
export const x = f(), { y } = o;
```

Reply:
Yes, and I think that is OK. 

There is no `DataFlow::Node` we can point to as "this node exported under the name `"y"`", so there is no natural result for `mod.getAnExportedValue("y")`. 

The closest we can come is the `"y"` property of the object  `o`, but we don't have a data-flow node for that.

Reply:
We should have a node for that. There is a `PropRead` corresponding to the property pattern `y` (see the `PropPatternAsPropRead` class), though I admit it may not be easy to get to. Something like this may be needed:
```codeql
DataFlow::lvalueNode(<BindingPattern>).getImmediatePredecessor()
```

Reply:
I found the node.
And I managed to get it in a way that doesn't look too bad.